sitewit/services.py

```diff
@@ -5,7 +5,10 @@
 from yoconfig import get_config
 
 import sitewit
-from sitewit.constants import CAMPAIGN_SERVICES
+from sitewit.constants import BillingTypes, CAMPAIGN_SERVICES, CampaignTypes
+
+
+_NEXT_CHARGE_FORMAT = '%Y-%m-%d 23:59:59'
```

Comment:
Maybe add **time** word to this name, otherwise it's unclear what is meant.

Reply:
Can you please suggest the name?  `nextCharge` is a field name on SW side (and a format is for that field).

Reply:
`_NEXT_CHARGE_DATETIME_FORMAT` ?

Reply:
`nextCharge` field is required to be a datetime string + it doesn't support any other formats. Won't the name `_NEXT_CHARGE_DATETIME_FORMAT` suggests that it might be some other format except for `DATETIME`?

Reply:
And doesn't it obvious from that string that it's a datetime format?

Reply:
counting 5 above comments i'd say following is quite a reasonable option:
let it just be original `_NEXT_CHARGE_FORMAT`.

Reply:
`next_charge_field_format`

Reply:
I'm fine with that.  Busy changing.